addons/l10n_ro/__openerp__.py

```diff
@@ -22,16 +24,26 @@
 {
     "name" : "Romania - Accounting",
     "version" : "1.0",
-    "author" : "ERPsystems Solutions",
+    "author" : "ERPsystems Solutions, Fekete Mihai, Tatár Attila",
     "website": "http://www.erpsystems.ro",
     "category" : "Localization/Account Charts",
     "depends" : ['account','account_chart','base_vat'],
     "description": """
-This is the module to manage the Accounting Chart, VAT structure, Fiscal Position and Tax Mapping.
```

Comment:
If not necessary (cf my two comments below on the main changes), we avoid changing module description as it breaks the existing translations.